### ISSUE TITLE
How to identify when video has enough data loaded to be played? oncanload, oncanloadthrough

### ISSUE THREAD

**Issue by lucashfreitas** [vimeo/player.js#628]
In some applications, we might want to do stuff only when the video is loaded, for that we can use `canplay` or `oncanplaythrough` events of video component. 

Vimeo player doesn't expose any of those browser API for video components. 

How do we identify if the video has enough downloaded data to be played?

Thanks.

**Comment by luwes** [comment]
yes, these events are not supported currently. have you tried using `player.getBuffered()`?
this method will return the buffered ranges the player has loaded so far.
